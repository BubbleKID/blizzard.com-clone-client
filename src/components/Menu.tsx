export default function Menu() {
    return (
        <div style={{height: '40px'}}>
            im menu
        </div>
    )
}