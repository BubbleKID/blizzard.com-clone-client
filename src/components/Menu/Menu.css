.menu-container {
    background-color: #0b0d12;
    border-bottom: solid 1px rgba(255,255,255,0.15);
}

.menu {
    height: 60px;
    margin: 0 auto;
    max-width: 1600px;
    display: flex;
    justify-content: space-between;
    align-items: baseline;
}

.menu-items {
    height: 60px;
    font-size: 16px;
}

.menu-items a {
    position: relative;
    display: inline-block;
    height: 60px;
    margin-right: 30px;
    text-transform: uppercase;
}

.menu-button {
    display: inline-block;
}

.menu-button span {
    cursor: pointer;
    display: inline-block;
    line-height: 60px;
}

.menu-logo {
    cursor: pointer;
    height: 60px;
    margin-right: 30px;
    fill: #009ae4;
}

.menu-logo svg{
    line-height: 22px;
    position: relative;
    top: 50%;
    transform: translateY(-50%);
    height: 34px;
}

.profile a {
    cursor: pointer;
    font-size: 17px;
    text-transform: capitalize;
    margin-right: 30px;
}