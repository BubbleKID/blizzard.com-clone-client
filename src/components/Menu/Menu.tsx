import './Menu.css';
import  { ReactComponent as MenuLogo } from '../../images/Blizzard_Entertainment-2.svg';
// import MenuLogo from '../../images/Blizzard_Entertainment-2.svg';

export default function Menu() {
    return (
        <div className="menu-container">
            <div className="menu">
                <div className="menu-items">
                    <a className="menu-logo"><MenuLogo /></a>
                    <div className="menu-button">
                        <a><span>Games</span></a>
                        <a><span>Shop</span></a>
                        <a><span>News</span></a>
                        <a><span>Esports</span></a>
                    </div>
                </div>
                <div className="profile">
                    <a><span>Support</span></a>
                    <a><span>My account</span></a>
                </div>
            </div>
        </div>
    )
}