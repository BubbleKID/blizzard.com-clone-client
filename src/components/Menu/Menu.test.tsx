import React from 'react';
import TestRenderer  from 'react-test-renderer';
import Menu from './Menu';

test('Menu should be rendered', () => {
    const component = TestRenderer.create(<Menu/>,);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});