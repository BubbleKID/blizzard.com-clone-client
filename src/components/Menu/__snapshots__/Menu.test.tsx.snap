// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should be rendered 1`] = `
<div
  className="menu-container"
>
  <div
    className="menu"
  >
    <div
      className="menu-items"
    >
      <a
        className="menu-logo"
      >
        <ReactComponent />
      </a>
      <div
        className="menu-button"
      >
        <a>
          <span>
            Games
          </span>
        </a>
        <a>
          <span>
            Shop
          </span>
        </a>
        <a>
          <span>
            News
          </span>
        </a>
        <a>
          <span>
            Esports
          </span>
        </a>
      </div>
    </div>
    <div
      className="profile"
    >
      <a>
        <span>
          Support
        </span>
      </a>
      <a>
        <span>
          My account
        </span>
      </a>
    </div>
  </div>
</div>
`;
